README.md

```diff
@@ -81,7 +81,24 @@ You will see AI analysis happen in the Terminal standard out.  The analysis will
 
 The easiest way to deploy this app is using the [Azure Dev CLI aka AZD](https://aka.ms/azd).  If you open this repo in GitHub CodeSpaces the AZD tooling is already preinstalled.
 
-To provision and deploy:
+1. Run the following command to initialize the project. 
+
+```bash
+azd init --template https://github.com/Azure-Samples/function-python-ai-textsummarize
```

Comment:
We should not do an init template in a git repo.  What we have today is more correct.